docs/howto/charm-to-python.rst

```diff
@@ -0,0 +1,126 @@
+.. _howto-migrate-to-python:
+
+Migrate from the ``charm`` plugin to the ``python`` plugin
+==========================================================
+
+The ``python`` Charmcraft plugin offers a faster, stricter means of packing an operator
```

Comment:
> Charmcraft plugin

Do we need to be this specific here? "Python Charmcraft plugin" tripped me up as a phrase.